cpp/ql/lib/semmle/code/cpp/Location.qll

```diff
@@ -73,8 +73,24 @@ class Location extends @location {
 
   /** Holds if `this` comes on a line strictly before `l`. */
   pragma[inline]
-  predicate isBefore(Location l) {
-    this.getFile() = l.getFile() and this.getEndLine() < l.getStartLine()
+  predicate isBefore(Location l) { isBefore(l, false) }
```

Comment:
It shows a warning about an implicit `this` here. Should that be fixed?

Reply:
Yeah, it really should. I'll add the explicit `this`, but I won't force-push the change since that will mess up the DCA run.